src/assets/scripts/game/GameClockView.js

```diff
@@ -0,0 +1,96 @@
+import $ from 'jquery';
+import { digits_integer } from '../utilities/radioUtilities';
+import { SELECTORS } from '../constants/selectors';
+import { TIME } from '../constants/globalConstants';
+
+/**
+ * @class GameClockView
+ */
+export default class GameClockView {
+    /**
+     * @for GameClockView
+     * @constructor
+     */
+    constructor() {
+        this.$element = null;
+        this.startTime = 0;
+        this.time = 0;
+
+        return this._init();
+    }
+
+    /**
+     * @for GameClockView
+     * @method _init
+     * @private
+     */
+    _init() {
+        this.$element = $(SELECTORS.DOM_SELECTORS.CLOCK);
+        this.$element.addClass(SELECTORS.CLASSNAMES.NOT_SELECTABLE);
```

Comment:
this would typically live in a `layout()` method.  this class is so small, it doesn't matter.  non-actionable and just an FYI.